parser_test.go

```diff
@@ -105,3 +110,49 @@ func TestDrawLabelAsPng(t *testing.T) {
 		})
 	}
 }
+
+func TestDrawLabelAsPng_Barcode128(t *testing.T) {
+	filenames := []string{
+		"barcode128_line_above",
+		"barcode128_line",
+		"barcode128_mode_a",
+		"barcode128_mode_d",
+		"barcode128_mode_n",
+		"barcode128_mode_u",
+		"barcode128_rotated",
+	}
+
+	for _, filename := range filenames {
+		t.Run(filename, func(t *testing.T) {
+			file, err := os.ReadFile("./testdata/" + filename + ".zpl")
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			expectedPng, err := os.ReadFile("./testdata/" + filename + ".png")
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			parser := NewParser()
+
+			res, err := parser.Parse(file)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			var buff bytes.Buffer
+
+			drawer := NewDrawer()
+
+			err = drawer.DrawLabelAsPng(res[0], &buff, drawers.DrawerOptions{})
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			if diff := cmp.Diff(buff.Bytes(), expectedPng); diff != "" {
+				t.Fatalf("unexpected output label, diff(+got,-want): %s", diff)
+			}
+		})
+	}
+}
```

Comment:
this test is not needed, right?

Reply:
Oh i thought I removed it already. It's removed now